package.json

```diff
@@ -84,7 +84,7 @@
     }
   ],
   "peerDependencies": {
-    "react": ">= 18"
+    "react": ">= 17"
```

Comment:
One other thing to watch out for is with the [Breadcrumbs component we use `useEffect`](https://github.com/chanzuckerberg/edu-design-system/blob/9998586a11ed82da7af2861417663e5961d12f01/src/components/Breadcrumbs/Breadcrumbs.tsx#L46) after the React 18 bump. Don't think it would have much problems with React 17 but something to keep an eye on